app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb

```diff
@@ -20,6 +20,7 @@ def index
     param :content_view_id, :identifier, :desc => N_("content view identifier"), :required => true
     param :name, String, :desc => N_("name of the puppet module")
     param :author, String, :desc => N_("author of the puppet module")
+    param :id, String, :desc => N_("the id of the puppet module to associate")
     param :uuid, String, :desc => N_("the uuid of the puppet module to associate")
```

Comment:
Thinking more about it, I don't know that this option would make sense so I am inclined to remove it (the 'id') option that I added since content view puppet modules are not connected to puppet modules at the database level.